charts/lily/templates/configmap-daemonconfig.yaml

```diff
@@ -39,4 +39,17 @@ data:
           Path = {{ .path  | quote }}
         {{- end }}
       {{- end }}
+    [Queue]
+      {{- if .Values.daemon.queue.asynq }}
```

Comment:
The pattern of including `enabled` in the `queue` map makes it easy to leave values and flag behavior on/off instead of commenting/changing structure.

Also, could `queue` be more descriptive. Maybe `cluster`?

So maybe something like `{{- if .Values.daemon.cluster.enabled }}`.

I would consider changing the config headers to match, too? 😅